demo/full/scripts/controllers/knobs/AudioTrack.jsx

```diff
@@ -14,6 +14,7 @@ const AudioTrackKnobBase = ({
   currentLanguage,
   availableLanguages = [],
 }) => {
+  debugger;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Remove this debugger statement. ([javascript:DebuggerStatement](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3ADebuggerStatement))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=278&issues=AWUjBS2pzFyjFVlut9x3&open=AWUjBS2pzFyjFVlut9x3)

[ISSUE_KEY]: AWUjBS2pzFyjFVlut9x3